### ISSUE TITLE
Failed test: simple/test-process-active-wraps.js 

### ISSUE THREAD

**Issue by jbergstroem** [joyent/node#8061]
For some reason, my system (node 37c2a528336b41b31abe4826b308071896e42423)  seems to count two ticks after starting the server:
```bash
./node test/simple/test-process-active-wraps.js 

assert.js:98
  throw new assert.AssertionError({
        ^
AssertionError: 2 == 1
    at expect (/home/jbergstroem/node/test/simple/test-process-active-wraps.js:28:10)
    at /home/jbergstroem/node/test/simple/test-process-active-wraps.js:37:3
    at Object.<anonymous> (/home/jbergstroem/node/test/simple/test-process-active-wraps.js:41:3)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:124:16)
    at node.js:811:3
```

built with: `./configure --prefix=/usr --shared-libuv --shared-http-parser --shared-zlib --shared-openssl --without-dtrace`, External libraries match the same versions as in deps (besides openssl since that is patched in deps).

**Comment by hefangshi** [comment]
got same test failed in v0.12, any information?

**Comment by mhdawson** [comment]
Just found this see https://github.com/joyent/node/pull/25326

**Comment by mhdawson** [comment]
I'll close this once #25236 goes in

**Comment by mhdawson** [comment]
25236 has landed in V0.12 head, closing
